examples/chronograf-ami/config/chronograf

```diff
@@ -0,0 +1,2 @@
+HOST=<__HOST__>
+PORT=<__PORT__>
```

Comment:
is there any point making sure these files have a newline at the end of them?

Reply:
During development, Chronograf weirdly complained about the newline so I removed it. Will do more tests as I continue on to the run module

Reply:
Ok, I was wondering 👍